packages/components/src/components/UtilityCard/UtilityCard.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { UtilityCard, InteractiveElementType } from "./UtilityCard";
+
+export default {
+  component: UtilityCard,
+  title: "Components/Utility Card",
+} as Meta;
+
+type Story = StoryFn<typeof UtilityCard>;
+
+const Template = (args: React.ComponentProps<typeof UtilityCard>) => {
+  return <UtilityCard {...args} />;
+};
+
+const href = "https://localyze.com";
+
+const onClick = (): void => window.location.assign(href);
```

Comment:
This is causing an issue that opens the URL on the same page of storybook. So, maybe it's better just using an `alert` to represent the click action.